Source Code/Ticket Management System/customer_menu.c

```diff
@@ -105,8 +113,42 @@ void viewFAQ()
 }
 
 void updateContactInfo()
-{
-	// TODO
+{	
+	system("CLS");
+	char email[512];
+	char OG_email[512];
+	wchar_t password[50];
+	wchar_t name[50];
+	wchar_t phone_number[20];	
+	struct User* logged_user=auth_getUser(); //fetchs the user from the container	
+	strcpy(OG_email,auth_getEmail()); //stores original Email of the user.
```

Comment:
Why do you need a copy of the email address? Why not work directly with `auth_getEmail()`?